pandas/core/ops/__init__.py

```diff
@@ -59,6 +59,37 @@
     rxor,
 )
 
+# -----------------------------------------------------------------------------
+# constants
+ARITHMETIC_BINOPS: Set[str] = {
```

Comment:
@jbrockmendel am I reinventing any wheels here? This seems like something useful enough that it would have been done before.

Reply:
i dont think so, no